crengine/src/lvdocview.cpp

```diff
@@ -6417,7 +6417,7 @@ void LVDocView::propsUpdateDefaults(CRPropRef props) {
 #endif
 	static int bool_options_def_true[] = { 1, 0 };
 	static int bool_options_def_false[] = { 0, 1 };
-	static int int_option_weight[] = { 100, 200, 300, 400, 500, 600, 700, 800, 900, 950 };
+	static int int_option_weight[] = { 100, 200, 300, 400, 450, 475, 500, 525, 550, 600, 650, 700, 800, 900, 950 };
```

Comment:
Feels strange to allow _some_ n25/50/75 in here but not others (why not 425 ? does it really make no change, even with large font sizes? and why it doesn't? some integer computation that would always cap it to the same value that 400 gives?)
I guess internally, the code should list all n25/50/75 (or use a min/max like you did recently for font sizes) - and if there's some visual/specific limitations, it might be in the frontend code to decide what reasonable values it wishes to expose.

(We have a fine tune number picker in KOReader where a user could pick 225. But this n25 stuff feels a bit too wild to me :) I'm not sure I want to expose this kind of granularity. Although I understand the wish of your user.)